builtin/providers/aws/resource_aws_wafregional_ipset.go

```diff
@@ -0,0 +1,162 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/waf"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsWafRegionalIPSet() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsWafRegionalIPSetCreate,
+		Read:   resourceAwsWafRegionalIPSetRead,
+		Update: resourceAwsWafRegionalIPSetUpdate,
+		Delete: resourceAwsWafRegionalIPSetDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"ip_set_descriptors": &schema.Schema{
```

Comment:
Similar to the other schema - it would make more sense to call this `ip_set_descriptor` IMO.